x-pack/solutions/observability/plugins/synthetics/e2e/synthetics/journeys/add_monitor.journey.ts

```diff
@@ -192,7 +192,8 @@ const createMonitorJourney = ({
           monitorType
         );
         expect(hasFailure).toBeFalsy();
-        await page.click('text=Update monitor');
+        await page.waitForTimeout(1000);
+        await page.getByTestId('syntheticsMonitorConfigSubmitButton').click();
```

Comment:
Does it mean the issue was trying to hit the submit button before it is active/visible?